packages/framework/src/Controller/Admin/PromoCodeController.php

```diff
@@ -59,6 +113,27 @@ public function listAction()
         ]);
     }
 
+    /**
+     * @Route("/promo-codes/list")
```

Comment:
Yesterday we were discussing it and we agreed that it would be better to keep current route and decide which site to display by some parameter. Why did you chose otherwise?

Reply:
Because if I would want to do it with GET parameter I would need to add new `Request` as a argument to the `listAction` and that would be a BC break

Reply:
Ok, I have discussed this with @PetrHeinz and we think that there should be some parameter in framework saying: useOldPromoCodes = true, this parameter would be now overwritten in project-base parameters saying useOldPromoCodes=false. So our current customers would have same behavior as before and new customers would have the new one. 

You then could use this parameter in menu to decide if to show old / new menu and in other parts (even inlineEdit in forms).